include/cuco/detail/trie/bit_vector/bit_vector.inl

```diff
@@ -0,0 +1,277 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace cuco {
+namespace experimental {
+
+template <class Allocator>
+bit_vector<Allocator>::bit_vector(Allocator const& allocator)
+  : words_{},
+    ranks_{},
+    ranks0_{},
+    selects_{},
+    selects0_{},
+    n_bits_(0),
+    allocator_(allocator),
+    aow_words_(nullptr),
+    aow_ranks_(nullptr),
+    aow_selects_(nullptr),
+    aow_ranks0_(nullptr),
+    aow_selects0_(nullptr)
```

Comment:
```suggestion
    n_bits_{0},
    allocator_[allocator},
    aow_words_{nullptr},
    aow_ranks_{nullptr},
    aow_selects_{nullptr},
    aow_ranks0_{nullptr},
    aow_selects0_{nullptr}
```
Always prefer curly braces over parentheses in initialization list

Reply:
005bd5d42177a3239e4b23545e9f328e7bb07633